packages/xchain-client/src/types.ts

```diff
@@ -88,11 +95,11 @@ export interface XChainClient {
   getExplorerTxUrl(txID: string): string
 
   validateAddress(address: string): boolean
-  getAddress(): Address
+  getAddress(index: number): Address
```

Comment:
Make `index` optional and use `0` as default value in client (as you have already done it at other places)

Reply:
i dont think that's possible here since this is the interface definition